ansible/README.md

```diff
@@ -249,6 +253,27 @@ In case one or more tasks fail or should not be run in the local environment, se
 ```bash
 ansible-playbook -b AdoptOpenJDK_Unix_Playbook/main.yml --skip-tags="install_zulu,jenkins_authorized_key,nagios_add_key,add_zeus_user_key"
 ```
+
+## Executing under vagrant (Windows)
+
+Ansible cannot be installed on a Windows machine, so you should boot a Linux VM using Vagrant and install it there instead:
```

Comment:
Do you have a Markdown linter in your IDE?  Might be worth running it, typically Markdown likes an empty newline after a `:` and there will be other things like that.

Reply:
Will do :)